FBSnapshotTestCaseDemo/Podfile.lock

```diff
@@ -13,8 +13,8 @@ EXTERNAL SOURCES:
     :path: ".."
 
 SPEC CHECKSUMS:
-  iOSSnapshotTestCase: 037b2e86ebc1cea90a48578c364295cd853c24aa
+  iOSSnapshotTestCase: 57355c72afe193439c624a4f3d150502093d0312
 
-PODFILE CHECKSUM: 23fc15db0c3233d57acaa5dbafbc9f5f89f2f477
+PODFILE CHECKSUM: a061ccaab7eb93ce08d3ae9762dd084a5d79c5ca
 
-COCOAPODS: 1.5.3
+COCOAPODS: 1.5.2
```

Comment:
Don't revert the cocoapods version

Reply:
Sorry this was incidental as my local version was 1.5.2